pkg/scheduledevents/scheduledevents.go

```diff
@@ -1,97 +1,50 @@
 package scheduledevents
```

Comment:
I believe this package name doesn't make sense anymore, neither the `ScheduledEvents` `struct`. Maybe something like `terminationdrainer`, `eventdrainer`? I don't have a good name.

Reply:
I created an interface called `EventHandler` and renamed this module as `DrainEventHandler`.
How does it sound?